### PR TITLE
✨ feat: Support TogetherAI as new model provider

### PR DESCRIPTION
**Pull request by phsontung**
#### 💻 变更类型 | Change Type

- [x] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 🔨 chore
- [ ] ⚡️ perf
- [ ] 📝 docs

#### 🔀 变更说明 | Description of Change

Support model provider: TogetherAI 

Screenshot from 2024-03-29 14-23-59
Screenshot from 2024-03-29 14-22-00

#### 📝 补充信息 | Additional Information
- Synced with main branch
- Update lobe-ui icons for TogetherAI

### CODE REVIEW

File: locales/tr-TR/setting.json
Comment (arvinxx) [inline locales/tr-TR/setting.json:1]: Nice catch! hhh

**Review by arvinxx — COMMENTED** [review]
LGTM, can you just rebase the code and wait for our work of lobe-icons?

**Comment by arvinxx** [comment]
@phsontung hello we have add the together icon: https://icons.lobehub.com/components/together

**Comment by phsontung** [comment]
> @phsontung hello, we have added the together icon: https://icons.lobehub.com/components/together

Thanks bro, I will update ASAP

**Comment by arvinxx** [comment]
@phsontung  the test is failed

**Review by arvinxx — APPROVED** [review]
LGTM now, I'll merge it! Thanks for your contributions~🎉

**Pull request merged**